lib/python/gladevcp/offsetpage.glade

```diff
@@ -38,252 +38,257 @@
     <data>
       <row>
         <col id="0" translatable="yes">Tool</col>
-        <col id="1" translatable="no">1</col>
-        <col id="2" translatable="no">2</col>
-        <col id="3" translatable="no">3</col>
-        <col id="4" translatable="no">4</col>
-        <col id="5" translatable="no">5</col>
-        <col id="6" translatable="no">6</col>
-        <col id="7" translatable="no">7</col>
-        <col id="8" translatable="no">8</col>
-        <col id="9" translatable="no">9</col>
+        <col id="1">0</col>
+        <col id="2">0</col>
+        <col id="3">0</col>
+        <col id="4">0</col>
+        <col id="5">0</col>
+        <col id="6">0</col>
+        <col id="7">0</col>
+        <col id="8">0</col>
+        <col id="9">0</col>
         <col id="10">True</col>
         <col id="11">False</col>
-        <col id="12" translatable="yes"></col>
-        <col id="13" translatable="no">black</col>
+        <col id="12">0</col>
+        <col id="13">black</col>
```

Comment:
I think this is the key why it didn't worked for me when I converted it. Have you done these changes manually?